main/tests.py

```diff
@@ -1,4 +1,33 @@
 """Tests are here."""
-# from django.test import TestCase
+from django.contrib.auth.models import User
+from django.test import TestCase
+from datetime import datetime
```

Comment:
According to most of conventions, the ordering must be a little different: stdlib packages are coming first, after that installed packages take place, finally current project's imports should be in the end of imports list.
`datetime` is an stdlib package.